server2/src/main/java/se/salt/server2/domain/assignment/models/AssignmentCategory.java

```diff
@@ -0,0 +1,6 @@
+package se.salt.server2.domain.assignment.models;
+
+public enum AssignmentCategory {
+    BACKEND,
+    FRONTEND
```

Comment:
The enum can hold the String also, so that you can avoid 1. having to use String.valueOf 2. The format will be off on the frontend (capitalized). So just do BACKEND("Backend") and add a private final field called value or whatever you see fit and then a constructor and getter.